docs/application-connector/docs/08-02-get-client-certificate.md

```diff
@@ -100,15 +102,49 @@ The response contains a valid client certificate signed by the Kyma Certificate
 
 After you receive the certificate, decode it and use it in your application. Register the services of your external solution through the Application Registry.
 
-## Call the Metadata and Event services on local deployment
+## Call the `metadata` endpoint
+
+You must call the `metadata` endpoint with the generated certificate to get URLs to the following:
+
+- the Application Registry API
+- the Events Service API
+- the `certificate renewal` endpoint
+- the `certificate revocation` endpoint
+
+Use the link you got in the second step to fetch the metadata information. Run:
```

Comment:
```suggestion
Use the link provided in the second step to fetch the metadata information. Run:
```
Do you mean this link `https://connector-service.kyma.local/v1/applications/signingRequests/info?token=h31IwJiLNjnbqIwTPnzLuNmFYsCZeUtVbUvYL2hVNh6kOqFlW9zkHnzxYFCpCExBZ_voGzUo6IVS_ExlZd4muQ==`?

Reply:
Done